src/store/reducers.ts

```diff
@@ -274,6 +274,7 @@ export interface SettingsType {
   locale: string;
   taskSettings: {
     filterBy: string | null;
+    sortBy: string;
```

Comment:
Please add a default value for it (a few lines above).

In addition, please add a settings migration, so it's always set. Look at `cacheMigrations` in `src/store/construct.ts`.

Reply:
Do I need to migrate values that don't matter, like `locale`?

Reply:
Nah. Take a look at the example I mentioned: https://github.com/etesync/etesync-web/blob/master/src/store/construct.ts#L154
You just need to do something like:
```
return {
  ...state,
  taskSettings: {
    ...state.taskSettings,
    sortBy: 'smart',
  },
};
```

Reply:
Just committed, let me know if I did it right.

Reply:
Looks right to me, and I also assume it works?

Reply:
Yep, works great!